apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -622,24 +654,29 @@ function initializeBlocklyWrapper(blocklyInstance) {
     blocksInWorkspace
       .filter(block => block.getParent() === null)
       .forEach(block => {
-        const svgTransformList = block.svgGroup_.transform.baseVal;
+        const svgTransformList = block.getSvgRoot().transform.baseVal;
```

Comment:
`getSvgRoot()` returns `svgGroup_`, which is a private field we should not be directly accessing.